### PR TITLE
Gitaly

### PR DESCRIPTION
**Pull request by IlyaMoskva**
For #162: The first version of Gitaly model.

### CODE REVIEW

File: reports/3pc-evaluation/Gitaly.tla [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+            either
+                rmState[self] := "prepared";
+                preparedNodes := preparedNodes \union {RM[self]};
+            or
+                rmState[self] := "aborted";
+                abortedNodes := abortedNodes \union {RM[self]};
+            end either;
+        Decide:
+            either
+                await Len(preparedNodes) * 2 > Len(RM) /\ rmState[1] = "prepared"
```
Comment (ChGen) [inline reports/3pc-evaluation/Gitaly.tla:47]: semicolon symbol is missing on the end of the line (translation error)

File: reports/3pc-evaluation/Gitaly.tla [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+                rmState[1] := "committed";
+            or
+                await Len(abortedNodes) * 2 > Len(RM)
```
Comment (ChGen) [inline reports/3pc-evaluation/Gitaly.tla:50]: semicolon symbol is missing on the end of the line (translation error)

**Review by ChGen — CHANGES_REQUESTED** [review]
Marked few issues in the code

File: reports/3pc-evaluation/Gitaly.tla [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+        Write:
+            await rmState[1] \in {"committed", "aborted"} /\ rmState[self] \notin {"committed", "aborted"};
+            rmState[self] := rmState[1];
+    end while;
+    print << "FINAL STATE:", rmState >>;
+end process;
+
+end algorithm;*)
+
+\* BEGIN TRANSLATION (chksum(pcal) = "44e2d299" /\ chksum(tla) = "ac2fab31")
```
[the remark this answers is not in the document]
Comment (IlyaMoskva) [reply reports/3pc-evaluation/Gitaly.tla]: fixed

**Review by g4s8 — APPROVED** [review]
@IlyaMoskva thanks

**Review by ivanbukhtiyarov — APPROVED** [review]
@ChGen check please again

**Comment by ivanbukhtiyarov** [comment]
Test comment

**Pull request merged**